ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/view_model/ArticleViewModel.java

```diff
@@ -31,7 +32,7 @@ public static ArticleViewModel getInstance(FragmentActivity activity) {
     public ArticleViewModel(@NonNull Application application) {
         super(application);
 
-        mArticleRepository = new ArticleRepository(application);
+        mArticleRepository = ArticleRoomDatabase.getInstance(application).articleRepository();
         mAttchReposistory = new AttachmentRepository(application);
```

Comment:
So it you would like for us to get repositories from the Database class instead of calling them directly.  That makes since so you know what database you are working with if add multiple databases.

Reply:
yeah, it also was necessary once I started annotating repositories as `@Dao` so that I could make entire methods `@Transactional`.